app/Filament/Resources/DatabaseHostResource.php

```diff
@@ -10,7 +10,7 @@ class DatabaseHostResource extends Resource
 {
     protected static ?string $model = DatabaseHost::class;
 
-    protected static ?string $label = 'Databases';
+    protected static ?string $label = 'Database Host';
```

Comment:
![image](https://github.com/user-attachments/assets/53bcf0e0-a427-4299-bb5f-57510d35f056)
With this change, you also need to edit the sidebar width.